test/rlx_tar_SUITE.erl

```diff
@@ -88,7 +76,7 @@ basic_tar(Config) ->
 
     %% only works in otp-21 and above
     case erlang:system_info(otp_release) of
-        R when R =:= "21" orelse R =:= "22" ->
+        R when R =:= "21" orelse R =:= "22" orelse R =:= "23" ->
```

Comment:
Should probably just convert the main part to an integer and compare for >= 21 at this point?

Reply:
Hah, yea, fixing.